sdcm/utils/sstable/sstable_utils.py

```diff
@@ -58,11 +60,18 @@ def is_sstable_encrypted(self, sstables=None) -> list:
             sstables = [sstables]
         sstables_encrypted_mapping = {}
         for sstable in sstables:
-            sstables_res = self.db_node.remoter.sudo(
-                f"{self.db_node.add_install_prefix('/usr/bin/scylla')} sstable dump-scylla-metadata"
-                "  --logger-log-level scylla-sstable=debug"
-                f" --sstables {sstable} --keyspace {self.keyspace} --table {self.table}",
-                ignore_status=True, verbose=True)
+            if ComparableScyllaVersion(self.db_node.scylla_version) >= '2023.2':
```

Comment:
This must be done out of the sstables loop.

Reply:
yes, I need to refactor this anyhow, since the logic is gonna be different, since we'll now gonna be ready the output of `scylla sstable`

Reply:
I've refactored it, to be on the top of the function